distro/configs/openmrs/initializer_config/patientidentifiertypes/msf_default_patientidentifiertypes.csv

```diff
@@ -1,6 +1,2 @@
 Uuid,Void/Retire,Name,Description,Required,Format,Format description,Validator,Location behavior,Uniqueness behavior
 05a29f94-c0ed-11e2-94be-8c13b969e334,,MSF ID,"MSF patient identifier, with check-digit",TRUE,,,,NOT_USED,UNIQUE
-22348099-3873-459e-a32e-d93b17eda533,,Legacy MSF ID,Identifier type to record optional previous identifers ,FALSE,,,,NOT_USED,
```

Comment:
Isn't the `Legacy MSF ID` the one used by DHIS2

Reply:
since the UUIDs are the same, maybe we can use the OpenMRS' and share an update that `Legacy MSF ID` has been renamed to `Legacy ID`
We can discuss more about this next week.

Reply:
@michaelbontyes is it okay to rename `Legacy MSF ID` to `Legacy ID` to match the naming from OpenMRS? or should we maintain this override?